winter/http/throttling.py

```diff
@@ -99,7 +96,8 @@ def create_throttle_classes(
 
         if getattr(throttling_annotation, 'rate', None) is not None:
             num_requests, duration = _parse_rate(throttling_annotation.rate)
-            throttling_ = Throttling(num_requests, duration, throttling_annotation.scope)
+            throttling_scope = throttling_annotation.scope or route.method.func.__qualname__
```

Comment:
```suggestion
            throttling_scope = throttling_annotation.scope or route.method.full_name
```
It seems that it is better

Reply:
Agree

Reply:
You could have pushed to button "Commit suggestion" and after that github would created commit (if it is convenient for you)